plinth/modules/bind/__init__.py

```diff
@@ -117,3 +113,29 @@ def diagnose():
     results.append(action_utils.diagnose_port_listening(53, 'udp6'))
 
     return results
+
+def default_config():
```

Comment:
For making any changes to the config, it needs to be handled in the action script. The reason is that Plinth will typically be running as a non-root user, and won't have permission to write to this file.

I recommend adding another parser in the action script, so we can run it like ```actions.superuser_run('bind', ['setup'])```.